packages/syncers/test/__snapshots__/DailyStatsSyncer.test.ts.snap

```diff
@@ -0,0 +1,211 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`DailyStatsSyncer > should aggregate data for all available days 1`] = `
+[
+  [
+    "2022-10-16",
+    "OTHER",
+    null,
+  ],
+  [
+    "2022-10-16",
+    "ROLLUP",
+    null,
+  ],
+  [
+    "2022-10-16",
+    null,
+    "OPTIMISM",
+  ],
+  [
+    "2022-10-16",
+    null,
+    null,
+  ],
+  [
+    "2023-05-10",
+    "ROLLUP",
+    null,
+  ],
+  [
+    "2023-05-10",
+    null,
+    "BASE",
+  ],
+  [
+    "2023-05-10",
+    null,
+    null,
+  ],
+  [
+    "2023-08-03",
+    "OTHER",
+    null,
+  ],
+  [
+    "2023-08-03",
+    null,
+    null,
+  ],
+  [
+    "2023-08-20",
+    "OTHER",
+    null,
+  ],
+  [
+    "2023-08-20",
+    "ROLLUP",
+    null,
+  ],
+  [
+    "2023-08-20",
+    null,
+    "ARBITRUM",
+  ],
+  [
+    "2023-08-20",
+    null,
+    "BASE",
+  ],
+  [
+    "2023-08-20",
+    null,
+    "OPTIMISM",
+  ],
+  [
+    "2023-08-20",
+    null,
+    null,
+  ],
+  [
+    "2023-08-28",
+    "OTHER",
+    null,
+  ],
+  [
+    "2023-08-28",
+    "ROLLUP",
+    null,
+  ],
+  [
+    "2023-08-28",
+    null,
+    "OPTIMISM",
+  ],
+  [
+    "2023-08-28",
+    null,
+    null,
+  ],
+]
+`;
+
+exports[`DailyStatsSyncer > should aggregate data for all available days 2`] = `
+[
+  "2022-10-16",
+  "2023-05-10",
+  "2023-08-03",
+  "2023-08-20",
+  "2023-08-28",
+]
+`;
+
+exports[`DailyStatsSyncer > should aggregate data for all available days 3`] = `
+[
+  [
+    "2022-10-16",
+    "OTHER",
+    null,
+  ],
+  [
+    "2022-10-16",
+    "ROLLUP",
+    null,
+  ],
+  [
+    "2022-10-16",
+    null,
+    "OPTIMISM",
+  ],
+  [
+    "2022-10-16",
+    null,
+    null,
+  ],
+  [
+    "2023-05-10",
+    "ROLLUP",
+    null,
```

Comment:
category is "ROLLUP" but rollup is null

Reply:
See [here](https://github.com/Blobscan/blobscan/pull/576#discussion_r1775177526)